website_sale_close/models/website.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019 Coop IT Easy SCRLfs
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp import fields, models
+
+
+class Website(models.Model):
+
+    _inherit = "website"
+
+    is_ecommerce_open = fields.Boolean(
+        string="Is e-commerce open",
+        default=False,
+        help="When checked e-commerce is open, when unchecked "
+        "e-commerce is closed.",
```

Comment:
`When checked, e-commerce is open. When unchecked, e-commerce is closed.`

Reply:
Ok, thanks for correction. :)

Reply:
Done.